contracts/staking/src/execute.rs

```diff
@@ -91,6 +91,14 @@ pub fn execute_liquid_stake(
         }
     );
 
+    // this handles a special case that through slashing and redeeming chaining we get into a state
```

Comment:
Is `chaining` typo? I don't really understand this special case. Can you elaborate? In what case will this case occur? The total liquid staking token (milkTIA) is zero and the total native token is not.

Reply:
`This scenario can happen if a slashing event occurs, the number of osmoTIA tokens is significantly reduced, and batches are submitted that cause the total_native_token and total_liquid_stake_token values to be asymmetrically reduced, resulting in total_liquid_stake_token to become zero while total_native_token is non-zero.`

Reply:
Even though we decided to also slash unstakers, so not sure how likely this is to happen.